lib/perf_hooks.js

```diff
@@ -471,6 +471,10 @@ class Performance extends PerformanceObserverEntryList {
     this[kClearEntry]('function', name);
   }
 
+  clearHttp2() {
+    this[kClearEntry]('http2');
+  }
```

Comment:
I do not think this path is viable, as we will add clearNet(), clearHttp(), clearHTTPS() and so on. How about clear(string)?